
# Calibartion parameters:
fx: 4.920642427973538133e+02
fy: 4.916400793457974032e+02
cx: 4.285082578952679455e+02
cy: 2.539114545212349014e+02
k1: -1.003137643876938212e-01
k2: 6.480104711529402484e-02
p1: 5.710408517258806684e-04
p2: 2.732927077291138779e-04
k3: -1.242539054818194745e-02

# Camera id to set image stream:
camera_id: 0

# Image pipeline to get the stream:
pipeline_id: 0
